src/components/dialog/style/lumapps/_index.scss

```diff
@@ -75,6 +75,7 @@
 
     &__progress-overlay {
         position: absolute;
+        z-index: 1000;
```

Comment:
You need to handle that case programmatically as there could be a dialog on top of an other -> check angularjs version